content/blog/e2e-design/index.md

```diff
@@ -174,9 +174,9 @@ trigger:
   url: localhost:9090/users
 
 verify:
-  - query: service ls
+  - query: swctl service ls
     expected: expected.services.yaml
-  - query: endpoint ls --service="YnVzaW5lc3Mtem9uZTo6cHJvamVjdEM=.1"
+  - query: swctl endpoint ls --service="YnVzaW5lc3Mtem9uZTo6cHJvamVjdEM=.1"
```

Comment:
How do you determine the validator? Such as file format. Do you need to say we support YAML format only for now?

Reply:
Added